content/cloud-computing/concepts/aws-management-console/aws-management-console.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: 'AWS Management Console'
+Description: 'The AWS Management Console is a web application that is home to all the various other consoles used to manage AWS services.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+The **AWS Management Console** is a web application that is home to all the various other consoles used to manage AWS services. The console home page is what's first seen when a user logs into AWS.
+The home page provides access to each service console and provides information needed to perform AWS-related tasks.
```

Comment:
Is it worth mentioning that the console is user customizable in this opening section? Since there are references to this in the bullet list it might be a good high level feature to mention in the main paragraph.